tests/expected_outputs/public/atss/atss_r50_fpn_1x.json

```diff
@@ -1,3 +1,3 @@
 {
-  "map": 0.493
```

Comment:
If this is a common value for both PyTorch and exported models, then it might be better to keep it as is.

Reply:
it has been changed because of keep_ratio. see the last commit. it is a common for both. but i'll change it one more time, i'll decrease number of test images because testing is so long

Reply:
I see. If testing setup is the same, then Ok.